src/test/java/space/obminyashka/items_exchange/service/MailServiceTest.java

```diff
@@ -45,9 +47,14 @@ void sendMail_shouldPassTheFlow() throws IOException {
         when(sendGrid.api(any())).thenReturn(new Response());
 
         final var emailTo = "test@mail.ua";
-        mailService.sendMail(emailTo, EmailType.REGISTRATION, Locale.ENGLISH, null);
+        UUID codeId = UUID.fromString("e58ed763-928c-4155-bee9-fdbaaadc15f3");
+        var endpointUrl = "https://obminyashka.space/api/v1/email/validate/" + codeId;
```

Comment:
It would be much better to construct that URL using constants or application variables

Reply:
Done